icecreamshop/icecreamshop/src/main/java/com/summerschool/icecreamshop/dto/RateDTO.java

```diff
@@ -0,0 +1,47 @@
+package com.summerschool.icecreamshop.dto;
+
+import com.summerschool.icecreamshop.model.Product;
+
+public class RateDTO {
+    private Long id;
+    private int rate;
```

Comment:
Can the rate be -1? Or 9999? We should have some validations in place for the rate value.